test/bin/scenario.sh

```diff
@@ -488,6 +492,12 @@ load_global_settings() {
     fi
 
     REDHAT_AUTHORIZED_KEYS="$(cat "${SSH_PUBLIC_KEY}")"
+    cat - <<EOF | tee -a "${HOME}/.ssh/config"
+Host 192.168.*
```

Comment:
This needs to be dynamically computed. Call get_vm_bridge_interface function and take X.Y. from the returned IP.